src/gravity/params.yaml

```diff
@@ -1,28 +1,142 @@
+    _description: "When to turn gravity off."
+    _default: "&infin;"
+    _units: "s"
+
+  uniform:
+    _type: node
+    _description: "Uniform graviational acceleration"
+    gx(1,2,3):
+      _type: Real
+      _description: "Uniform gravity: acceleration in the x(1,2,3)-dir."
+      _units: "cm s^-2"
+
+  point:
+    _type: node
+    _description: "Point mass gravity."
+    gm:
+      _type: Real
+      _description: "Normalization parameter setting GM and the mass of the body."
+      _units: "dyn cm^2 g^-1"
+    soft:
+      _type: Real
+      _description: "Plummer softening radius of the body"
+      _default: "0.0"
+      _units: "cm"
+    sink:
+      _type: Real
+      _description: "Accretion radius of the body. Material inside this distance to the point is gradually removed"
+      _default: "0.0"
+      _units: "cm"
+    sink_rate:
+      _type: Real
+      _description: "Mass removal rate."
+      _default: "0.0"
+      _units: "s^-1"
+    x:
+      _type: Real
+      _description: "X position"
+      _default: "0.0"
+      _units: "cm"
+    y:
+      _type: Real
+      _description: "Y position"
+      _default: "0.0"
+      _units: "cm"
+    z:
+      _type: Real
+      _description: "Z position"
+      _default: "0.0"
+      _units: "cm"
+
+  binary:
+    _type: node
+    _description: "Binary mass gravity."
+    gm:
+      _type: Real
+      _description: "Normalization parameter setting GM and the total mass of the binary"
+      _units: "dyn cm^2 g^-1"
+    soft(1,2):
+      _type: Real
+      _description: "Plummer softening radius of the primary/secondary"
+      _default: "0.0"
+      _units: "cm"
+    sink(1,2):
+      _type: Real
+      _description: "Accretion radius of the primary/secondary. Material inside this distance to the point is gradually removed"
+      _default: "0.0"
+      _units: "cm"
+    sink_rate(1,2):
+      _type: Real
+      _description: "Mass removal rate of the primary/secondary."
+      _default: "0.0"
+      _units: "s^-1"
+    x:
+      _type: Real
+      _description: "X position of the center of mass"
+      _default: "0.0"
+      _units: "cm"
+    y:
+      _type: Real
+      _description: "Y position of the center of mass"
+      _default: "0.0"
+      _units: "cm"
+    z:
+      _type: Real
+      _description: "Z position of the center of mass"
+      _default: "0.0"
+      _units: "cm"
+    q:
+      _type: Real
+      _description: "Binary mass ratio"
+    a:
+      _type: Real
+      _description: "Binary semi-major axis"
+      _units: "cm"
+    e:
+      _type: Real
+      _description: "Binary eccentricity"
+      _default: "0.0"
+    i:
+      _type: Real
+      _description: "Binary inclination (deg)."
+      _default: "0.0"
+      _units: "deg"
+    omega:
+      _type: Real
+      _description: "Binary argument of pericenter (deg)."
+      _default: "0.0"
+      _units: "deg"
+    Omega:
+      _type: Real
+      _description: "Binary longitude of ascending node (deg)."
```

Comment:
Do we want to keep the `(deg)` if we have `_units`?

Reply:
I kind of what to doubly emphasize to put it in degrees

Reply:
I have a medium preference to not double emphasize units, but let's defer this discussion to when we have units working in the code, and maybe unit conversions as an option in the input parameter files, and then it may be clearer what the best practices for writing descriptions will be.